### PR TITLE
Add extension fields for package factory

### PR DESCRIPTION
**Pull request by mezzode**
Attempt at adding package factory to repo JSON for tachiyomiorg/website#961. Since this is only a CI change I haven't done most of the checklist

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: .github/scripts/create-repo.sh [hunk 1]
```diff
@@ -55,6 +56,7 @@ for APK in ${APKS[@]}; do
         --argjson nsfw $NSFW \
         --argjson hasReadme $HASREADME \
         --argjson hasChangelog $HASCHANGELOG \
+        --argjson pkgFactory $PKGFACTORY \
```
Comment (ghostbear) [inline .github/scripts/create-repo.sh:59]: You have to add this to the JSON template too

Comment (mezzode) [reply .github/scripts/create-repo.sh:59]: Oops fixed, can't believe I missed that

**Comment by arkon** [comment]
This actually fails btw (I tried it on my fork):

```
jq: invalid JSON text passed to --argjson
Use jq --help for help with command-line options,
or see the jq manpage, or online docs  at https://stedolan.github.io/jq
[]
```

It's a bit difficult to test though, you'd need to fiddle around with the script locally.

**Comment by mezzode** [comment]
Fixed it, was using the wrong option for parsing strings and wasn't handling empty strings properly

**Comment by arkon** [comment]
- [Test run](https://github.com/arkon/tachiyomi-extensions/actions/runs/3357319376/jobs/5563074525)
- [Resulting file](https://github.com/arkon/tachiyomi-extensions/blob/5399453b23889a1ff7ec12c8fb0db70600ccfb95/index.json)

Might have to do with potentially empty `tachiyomi.extension.factory` values?

**Comment by alessandrojean** [comment]
I can't add the `hacktoberfest-accepted` label on the website repository as I don't have write permissions there, but added here.

**Comment by arkon** [comment]
> Could this have been an upstream issue or flakiness, does it still happen if you rerun the workflow with latest master merged?

It's on top of master, so I don't _think_ so?

> I tried testing that myself on my fork but I can't run the full CI pipeline because I don't have the signing key for building the APKs,

You could try either just setting your own key or tweaking the workflows to not require it.

**Comment by alessandrojean** [comment]
Are you still willing to work on this/fix the errors? Otherwise, I will just close it for now.

**Comment by alessandrojean** [comment]
Closing as it seems inactive.

**Pull request closed without merging**